nikola/utils.py

```diff
@@ -1255,6 +1255,25 @@ class ExtendedRSS2(rss.RSS2):
 
     xsl_stylesheet_href = None
 
+    def __init__(self, itunes_author=None, itunes_summary=None,
+                 itunes_name=None, itunes_email=None,
+                 itunes_image=None, itunes_categories=None,
+                 itunes_explicit=None, **kwargs):
+        self.itunes_author = itunes_author
+        self.itunes_summary = itunes_summary
+        self.itunes_name = itunes_name
+        self.itunes_email = itunes_email
+        self.itunes_image = itunes_image
+        self.itunes_categories = itunes_categories
+        self.itunes_explicit = itunes_explicit
+
+        # It's an old style class
+        rss.RSS2.__init__(self, **kwargs)
+
+    def _itunes_attributes (self):
```

Comment:
Oops. That's a minor stylistic affectation I use in my own code, but don't intend to inflict on anyone. Just a habit. :)